### PR TITLE
Create welcome screen

### LINKED ISSUE #6: Create Welcome Screen
## REQUIREMENTS / DESCRIPTION

The welcome screen is the first interface that users will see when opening the application. Its purpose is to capture the user's attention, convey the identity of the application and provide an attractive introduction. 

It must contain the following requirements:

- [x] Logo
- [x] Slogan
- [x] Terms and conditions
- [x] Button get started

## DESIGN

image

### PR DESCRIPTION
**Pull request by montsegv-2**
# Description

The welcome screen is usually the first thing users see when they open an app, and it sets the tone for the user experience. A well-designed welcome screen can make a positive first impression and encourage users to explore further. It should be visually appealing, informative, and user-friendly.

What did you implemented/Why did you implemented this:

- Implemented a design for welcome screen.
- Added logo and slogan for the app foodiecare 
- A primary button component and navigation to login screen

## Screenshots

Simulator Screenshot - iPhone 14 Plus - 2023-08-01 at 18 35 24

### CODE REVIEW

**Review by landillo — APPROVED** [review]
LGTM

**Review by mcmmiguel — APPROVED** [review]
Good job!

**Review by AlbertoG22 — APPROVED** [review]
Nice

File: src/components/ButtonPrimary/ButtonPrimary.tsx [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
 interface ButtonProps {
-    title: string;
-    onPress?: () => void;
-    isValid?: boolean;
-    imgSource?: any;
-  }
+  title: string;
+  onPress?: () => void;
+  isValid?: boolean;
+  imgSource?: any;
```
Comment (wiltrouble) [inline src/components/ButtonPrimary/ButtonPrimary.tsx]: Image source should be a string type

**Review by wiltrouble — CHANGES_REQUESTED** [review]

File: src/components/ButtonPrimary/ButtonPrimary.tsx (hunk 1 above)
Comment (montsegv-2) [reply src/components/ButtonPrimary/ButtonPrimary.tsx]: Done

**Review by wiltrouble — APPROVED** [review]

**Pull request merged**